README.md

```diff
@@ -21,7 +21,9 @@ file_in,                                   Mandatory parameter.Directory path to
 -d FILEDIFF, --fileDiff FILEDIFF           Writes the difference matrix into a file.
 -m METRICSFILE, --metricsFile METRICSFILE  CSV file containing metrics definition. Every line contains 4 elements: metric_name,file_extension,command_to_run,output_file_name
 -e EXCLUDEFOLDERS, --excludeFolders        The path to the file containing the folders and files that should be excluded from creating checksums.
--k CHECKCORRUPTION, --checkCorruption      The script verifies if any files are corrupted ,when this flag is set as true 
+-k CHECKCORRUPTION, --checkCorruption      The script verifies if any files are corrupted ,when this flag is set as true
```

Comment:
Some flags are missing in the README. For instance `--checksumFile` but maybe others. Both should be kept in sync, just dump the output of `-h` in the README periodically.